mobile/src/main/java/org/openhab/habdroid/ui/WidgetAdapter.kt

```diff
@@ -261,7 +265,8 @@ class WidgetAdapter(
             colorMapper,
             serverFlags,
             chartTheme,
-            { widgetsByParentId[widget.id] }
+            { widgetsByParentId[widget.id] },
```

Comment:
I'm still new to Kotlin and I don't understand why this one needs to be enclosed with braces, vs the one I've just added below doesn't (seem to) need braces. The line above was suggested by @maniac103

Reply:
This one is a function, while the one below isn't. It was just meant as an optimization so the list didn't need to be copied prematurely.